src/DatabaseSchema.php

```diff
@@ -27,5 +27,7 @@ public function __construct($version = 0.1)
         parent::__construct($version);
 
         $this->addModel('User', User::class);
+        $this->addModel('UserPastPassword', UserPastPassword::class);
+        $this->addModel('UserLoginAttempt', UserLoginAttempt::class);
```

Comment:
I will chat to you about this one - there is a need to get wider user auditing in place for ProjectBlue - however I'm not sure how something 'generic' is plugged into a specific module like this one. Perhaps it will be a more general name of tblAuthenticationUserAudit, a string 'type', a 'data' field and then we can store password changes, log in and log outs, and other actions like 'user changed this' kind of stuff.